dandi/publish/views/stats.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import User
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from dandi.publish.models import Asset, Dandiset
+
+
+@api_view()
+def stats_view(self):
+    dandiset_count = Dandiset.objects.count()
+    published_dandiset_count = Dandiset.published_count()
+    user_count = User.objects.count()
+    size = Asset.total_size()
+    return Response(
+        {
+            'dandiset_count': dandiset_count,
+            'published_dandiset_count': published_dandiset_count,
+            'user_count': user_count,
+            'size': size,
```

Comment:
I don't see a reason not to include `species_count`, `subject_count`, and `cell_count`. They were explicitly requested and will be added at some point, and are already present on the front page. I think leaving the scaffolding will save time later on, even if they are just placeholders for now.